### DISCUSSION TITLE
orm_mode ignored (when using with FastAPI)?

### DISCUSSION THREAD

**Discussion by JustinGuese** [tiangolo/pydantic-sqlalchemy#93] [Questions] [answered]
looking at the script, I would expect orm_mode to be turned on by default, but when using the model I am getting the usual "AttributeError: 'dict' object has no attribute 'email'"?

```
class User(Base):
    __tablename__ = "users"
    id = Column(Integer, primary_key=True, index=True, autoincrement=True, unique=True)
    email = Column(String, index=True, unique=True)
    organization_id = Column(Integer, ForeignKey("organizations.id"), index=True, nullable=True)
    role = Column(ENUM(Role), nullable=False, default=Role.ADMIN)

UserPD = sqlalchemy_to_pydantic(User)

[...]

@app.get("/users")
def getUsers(db: Session = Depends(get_db)): -> List[UserPD]
    users = db.query(User).all()
    users = [UserPD.from_orm(u) for u in users]
    # this one would fail
    print(users[0].email)
    # this one would work
    print(users[0]["email"]
```

**Comment by tiangolo** [comment, marked as answer]
Thanks for the patience with my reply! :sweat_smile:

I deprecated this library a while ago in favor of SQLModel which would solve all the same use cases in a much better way, and I'm directing all the effort in that direction.

SQLModel uses Pydantic models to define both data and table (SQL) models, so it's easier to do all these things (and there's no need to use `orm_mode` as it comes by default). :rocket:
